ingest/src/main/java/com/connexta/ingest/TransformException.java

```diff
@@ -0,0 +1,4 @@
+package com.connexta.ingest;
+
+public class TransformException extends Exception {
```

Comment:
✏️ TransformException sounds like this is thrown when is failed to be transformed, but it really means that we weren't able to submit the transform request. I suggest renaming to IngestException or something else